Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -2200,7 +2200,7 @@ void WebPage::loadRequest(LoadParameters&& loadParameters)
 
     localFrame->loader().setHTTPFallbackInProgress(loadParameters.isPerformingHTTPFallback);
     localFrame->loader().setRequiredCookiesVersion(loadParameters.requiredCookiesVersion);
-    localFrame->loader().load(WTFMove(frameLoadRequest));
+    localFrame->loader().load(WTFMove(frameLoadRequest), loadParameters.requester);
```

Comment:
You can use `WTFMove(loadParameters.requester)` here which I think means you can move the request all the way in to the DocumentLoader to avoid any copying.